catalogs/catalog-hive/build.gradle.kts

```diff
@@ -118,3 +152,62 @@ tasks {
     dependsOn(copyCatalogConfig, copyCatalogLibs)
   }
 }
+
+tasks.test {
+  val skipITs = project.hasProperty("skipITs")
+  if (skipITs) {
+    exclude("**/docker/**")
```

Comment:
Why above in hadoop catalog it is `**/integration/**`, here is "**/docker/**"?

Reply:
I have changed it to `e2e` currently, there is something wrong with HiveIT tests if we use `integration` here, it's weird enough and I can't find the root cause.

Reply:
All have been resolved except for this. Please take a look @jerryshao